build/changelog/ChangelogBuilder.php

```diff
@@ -0,0 +1,149 @@
+<?php
+namespace Aws\Build\Changelog;
+
+/**
+ * @internal
+ */
+
+class ChangelogBuilder 
+{
+  
+  public $newServiceFlag = False;
+
```

Comment:
Instead putting all methods into this `ChangelogBuilder` with same order as script file, I'm actually suggesting having a `ChangelogBuilder` class that could be used to initialize `ChangelogBuilder` objects, perhaps a class that takes directory parameter or some option array for constructor, and then have some method to be called. You can take a look at other builder class we have in the repo, for example [this](https://github.com/aws/aws-sdk-php/blob/master/build/ServiceBuilder.php).